### PR TITLE
Render pretty summaries

### PR DESCRIPTION
**Pull request by woodruffw**
WIP.

### CODE REVIEW

**Comment by woodruffw** [comment]
https://github.com/trailofbits/x509-limbo/actions/runs/6645470148 shows how this renders.

File: harness/render-summary.py [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+_render(f"## Limbo results for `{results["harness"]}`")
+
+for result in results["results"]:
+    testcase_id = result["id"]
+    actual_result = result["actual_result"]
+    context = result["context"]
+    if not context:
+        # Normalize missing context into an empty string.
+        context = ""
```
Comment (tnytown) [inline harness/render-summary.py:47]: Nit: is this working as expected? I see a few `Additional context: None` lines in the run you linked for `openssl`.

**Review by tnytown — APPROVED** [review]
Looks nice!

File: harness/render-summary.py (hunk 1 above)
Comment (woodruffw) [reply harness/render-summary.py:47]: That run is from before I pushed up that change; from a quick look, the latest render doesn't have those `None`s anymore 🙂

**Pull request merged**